internal/cmd/operator-sdk/scorecard/cmd.go

```diff
@@ -142,8 +142,7 @@ func (c *scorecardCmd) convertXunit(output v1alpha3.TestList) xunit.TestSuites {
 			}
 			tSuite.TestCases = append(tSuite.TestCases, tCase)
 			tSuite.URL = item.Spec.Image
-			//TODO: Add TestStuite ID when API updates version
-			//tSuite.ID = item.Spec.UniqueID
+			tSuite.ID = item.Spec.UniqueID
```

Comment:
What happens if `item.Spec.UnqiueID` is unset? Does `tSuite.ID` stay empty?

Reply:
Just ran it with UniqueID unset to confirm, tSuite.ID stays empty.

Reply:
Well this should probably have a default (random) value if not configured with one right?

Reply:
So the only users of this field are folks using this for xunit, and in those scenarios the ID's are being fed in the config anyways and there aren't really other use cases for this value (since the testcase names themselves are also unique).

Reply:
Will leaving the ID field of XUnit output empty cause errors when an XUnit parser attempts to parse this stuff? If so we should either set this to the testcase name, some random string, or (probably better) error out.

Reply:
That's a good question I will find out. If it would cause errors, I agree with the solution of erroring out if the field is left empty. I am not opposed to setting the field to the testcase name if we absolute must populate it, as opposed to the current omit if empty behavior.

Reply:
Having an empty UID field does not _appear_ to cause issues in the polarion importer

Reply:
Is this an optional field that the user can set or leave it unset?

Reply:
Also, would this need to conform to any naming conventions?
Will the XUnit parser work for all special characters?